spec/javascripts/modules/remove-filter-spec.js

```diff
@@ -79,6 +79,7 @@ describe('remove-filter', function () {
   })
 
   afterEach(function () {
+    $('.js-remove').remove()
```

Comment:
Not familiar with the filter, so take my suggestion with a pinch of salt, but could we remove the whole `$facets` from the DOM, instead of one element at a time?

Reply:
Yes, I wanted to do that, but I tried `$facets.remove()` and it throws a `$facets.remove is not a function` error for every test. I don't know why that doesn't work, I thought it should do. Previously the test wasn't cleaning up after itself at all.